go/engine/id_test.go

```diff
@@ -177,6 +170,7 @@ type FakeIdentifyUI struct {
 	Outcome         *keybase1.IdentifyOutcome
 	StartCount      int
 	Token           keybase1.TrackToken
+	BrokenTracking  bool
```

Comment:
the `FakeIdentifyUI` keeps track of if there was a `BreaksTracking` bool or not...